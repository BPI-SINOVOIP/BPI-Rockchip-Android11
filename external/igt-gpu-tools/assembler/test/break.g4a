/* The break instruction syntax, which is currently just what was in the BNF,
 * is bad.  It really needs 2 arguments -- pop count (19:16, how many
 * loops to break out of), and the IP count (15:0).  For now, this argument
 * should cover 1 loop, and jumping 2 instructions.
 */
break 65538;
