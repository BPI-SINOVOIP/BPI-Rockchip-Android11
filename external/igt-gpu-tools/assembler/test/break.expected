   { 0x00000028, 0x34001c00, 0x00011400, 0x00010002 },
